rpc/lib/client/http_client.go

```diff
@@ -74,6 +74,7 @@ func makeHTTPClient(remoteAddr string) (string, *http.Client) {
 	protocol, address, dialer := makeHTTPDialer(remoteAddr)
 	return protocol + "://" + address, &http.Client{
 		Transport: &http.Transport{
+			DisableCompression: true,
```

Comment:
Could you please explain why we should not use a default here?

Reply:
sure. I have just finish writing the description of this pr. please have a check